actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java

```diff
@@ -175,6 +184,60 @@ private void updateUser(Request actorMessage) {
     TelemetryUtil.telemetryProcessingCall(userMap, targetObject, correlatedObject);
   }
 
+  private void updateUserOrganisations(Request actorMessage) {
+    if (null != actorMessage.getRequest().get(JsonKey.ORGANISATIONS)) {
+      ProjectLogger.log(
+          "UserManagementActor:updateUserOrganisations : updateUserOrganisation Called with valide data",
```

Comment:
updateUserOrganisation Called with valide data -> called

Rest of words are redundant